presentation/src/main/java/com/yapp/growth/presentation/ui/main/manage/ManageScreen.kt

```diff
@@ -0,0 +1,366 @@
+        backgroundColor = Color.Transparent,
+        indicator = @Composable { tabPositions ->
+            TabRowDefaults.Indicator(
+                modifier = Modifier
+                    .tabIndicatorOffset(tabPositions[tabIndex])
+                    .padding(horizontal = 36.dp),
+                color = Gray900
+            )
+        },
+        divider = @Composable { TabRowDefaults.Divider(color = Gray200) }
+    ) {
+        ManageTapMenu.values().forEachIndexed { index, menu ->
+            val planCount = when (menu) {
+                ManageTapMenu.WAITING_PLAN -> waitingPlans.size
+                ManageTapMenu.FIXED_PLAN -> fixedPlans.size
+            }
+
+            Tab(
+                selected = tabIndex == index,
+                onClick = { onTabClick(index) },
+                text = {
+                    Text(
+                        text = stringResource(id = menu.textId) + "($planCount)",
+                        style = PlanzTypography.subtitle1,
+                        color = Color.Unspecified
+                    )
+                },
+                selectedContentColor = Gray900,
+                unselectedContentColor = Gray300
+            )
+        }
+    }
+}
+
+@OptIn(ExperimentalPagerApi::class)
+@Composable
+fun ManagePager(
+    pagerState: PagerState,
+    waitingPlans: List<Plan.WaitingPlan>,
+    fixedPlans: List<Plan.FixedPlan>,
+    onWaitingItemClick: (Int) -> Unit,
+    onFixedItemClick: (Int) -> Unit,
+) {
+    HorizontalPager(
+        state = pagerState,
+        modifier = Modifier.fillMaxSize()
+    ) { index ->
+        LazyColumn(
+            modifier = Modifier.fillMaxSize(),
+            verticalArrangement = Arrangement.spacedBy(12.dp)
+        ) {
+            item {
+                Spacer(modifier = Modifier.height(24.dp))
+            }
+
+            item {
+                when (index) {
+                    ManageTapMenu.WAITING_PLAN.ordinal -> {
+                        ManagePlansList(
+                            plans = waitingPlans,
+                            type = ManageTapMenu.WAITING_PLAN,
+                            onItemClick = onWaitingItemClick
+                        )
+                    }
+                    ManageTapMenu.FIXED_PLAN.ordinal -> {
+                        ManagePlansList(
+                            plans = fixedPlans,
+                            type = ManageTapMenu.FIXED_PLAN,
+                            onItemClick = onFixedItemClick
+                        )
+                    }
+                }
+            }
+        }
+    }
+}
+
+@Composable
+fun ManagePlansList(
+    plans: List<Plan>,
+    type: ManageTapMenu,
+    onItemClick: (Int) -> Unit,
+) {
+    Column(
+        modifier = Modifier.padding(horizontal = 20.dp), /* TODO: 스크롤 범위 수정 */
+        verticalArrangement = Arrangement.spacedBy(12.dp)
+    ) {
+        plans.forEach { plan ->
+            ManagePlansItem(plan = plan, type = type, onItemClick = onItemClick)
```

Comment:
```suggestion
      items(plans.size) { index ->
          ManagePlansItem(plan = plansList[index], type = type, onItemClick = onItemClick)
          
      or
      
      itemsIndexed(plansList) { index, _ ->
          ManagePlansItem(plan = plansList[index], type = type, onItemClick = onItemClick)
      
```

LazyColumn 사용 가능해 보이느데 혹시 Column 을 사용하신 이유가 있다면 알려주시면 감사하겠습니다 ㅎㅎ

Reply:
위의 코멘트와 동일해요!
인덱스를 굳이 사용하지 않아도 되기 때문에 forEachIndex보다는 forEach가 적절하다고 판단하였고, 이미 LazyColumn 스코프 안에서 호출되는 컴포저블 함수입니당

Reply:
위의 코멘트와 동일해요! 인덱스를 굳이 사용하지 않아도 되기 때문에 forEachIndex보다는 forEach가 적절하다고 판단하였고, 이미 LazyColumn 스코프 안에서 호출되는 컴포저블 함수입니당

Reply:
음 요 부분이 조금 혼란을 줄 수 있는 부분인 것 같은데,, 고러면 아예 저 친구를 아이템째로(`item{ // }`) 빼서 LazyColoumScope로 명시해 주는 게 좋을지 고민이네요..!

Reply:
HorizontalPager 의 contents 부분을 Column 혹은 Box 로 구성하고 Spacer(padding top) 준 다음 
`            when (index) {
                ManageTapMenu.WAITING_PLAN.ordinal -> {
                    ManagePlansList(
                        plans = waitingPlans,
                        type = ManageTapMenu.WAITING_PLAN,
                        onItemClick = onWaitingItemClick
                    )
                }
                ManageTapMenu.FIXED_PLAN.ordinal -> {
                    ManagePlansList(
                        plans = fixedPlans,
                        type = ManageTapMenu.FIXED_PLAN,
                        onItemClick = onFixedItemClick
                    )
                }
            }`

이게 들어가고, ManagePlansList 에서 LazyColumn 으로 구성하는건 어떤가요?